src/server/http/__tests__/xsrf.js

```diff
@@ -29,10 +30,31 @@ describe('xsrf request filter', function () {
 
     await kbnServer.ready();
 
-    const routeMethods = nonDestructiveMethods.filter(method => method !== 'HEAD').concat(destructiveMethods);
```

Comment:
The only thing I can think of is that by pure coincidence, all of our bulk json requests go directly through the proxy to Elasticsearch, so perhaps that isn't enforcing the same restrictions as discrete HTTP endpoints?